flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -111,18 +112,20 @@ public void testMetricReporting() throws Exception {
 
 			verify(postRequestedFor(urlPathEqualTo("/write"))
 				.withQueryParam("db", equalTo(TEST_INFLUXDB_DB))
+				.withQueryParam("rp", equalTo(retentionPolicy))
 				.withHeader("Content-Type", containing("text/plain"))
 				.withRequestBody(containing("taskmanager_" + metricName + ",host=" + METRIC_HOSTNAME + ",tm_id=" + METRIC_TM_ID + " count=42i")));
 		} finally {
 			metricRegistry.shutdown().get();
 		}
 	}
 
-	private MetricRegistryImpl createMetricRegistry() {
+	private MetricRegistryImpl createMetricRegistry(String retentionPolicy) {
 		MetricConfig metricConfig = new MetricConfig();
 		metricConfig.setProperty("host", "localhost");
 		metricConfig.setProperty("port", String.valueOf(wireMockRule.port()));
 		metricConfig.setProperty("db", TEST_INFLUXDB_DB);
+		metricConfig.setProperty("retentionPolicy", retentionPolicy);
```

Comment:
Actually, it would be nicer to have all these config options use the keys from the `InfluxdbReporterOptions`, e.g. `HOST.key()` or `RETENTION_POLICY.key()` so that they are up-to date if options are renamed/deprecated.

Reply:
Corrected based on recommendations.